CMakeLists.txt

```diff
@@ -71,6 +71,10 @@ if ("${K4A_ENABLE_LEAK_DETECTION_CMAKE}" STREQUAL "1")
     add_definitions(-DK4A_ENABLE_LEAK_DETECTION)
 endif()
 
+#
```

Comment:
Comment to to say RPATH of origin results in dlopen searching for binaries in the local directory first?

Reply:
Whoops, good call